index.html

```diff
@@ -72,6 +72,9 @@ <h1 class="title">HERE Maps on GitHub</h1>
 				client_secret : '918cbdf3a49ba292ff5e51e01e893ae0117c93ac'
 				},
 				repos = [
+					{id: 'mobility-on-demand-use-cases-ios',
+					type: 'Objective-C',
+					description:'HERE mobility on-demand demo written as an iOS application'},
```

Comment:
Hmm, so above we say "use-cases" (plural), here we say "demo" (singular), and we used to call them "examples". What should we align on here?